lib/features/chat/domain/helix_get_channel_emotes.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+import 'helix_images.dart';
+
+part 'helix_get_channel_emotes.freezed.dart';
+part 'helix_get_channel_emotes.g.dart';
+
+@freezed
+class HelixGetChannelEmotesResponse with _$HelixGetChannelEmotesResponse {
+  const factory HelixGetChannelEmotesResponse({
+    required int total,
```

Comment:
I dont see total in the documentation https://dev.twitch.tv/docs/api/reference#get-channel-emotes